scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -240,6 +268,9 @@ task M2 {
 
   output {
     File output_vcf = "${output_vcf_name}.vcf"
+    File output_bamOut = "bamout.bam"
+    String tumor_bam_sample_name = read_lines("tumor_name.txt")[0]
```

Comment:
You can use `readString` to read a single line

Reply:
Done

Reply:
`read_string`